vagrant/Makefile

```
@@ -21,6 +21,9 @@ setup.test.only:
 setup.cluster.only:
 	@$(VSSH) setup "make -C /home/vagrant/ansible setup.cluster"
 
-setup.cluster: setup.prep setup.test.only setup.cluster.only
+client1.test:
+	@$(VSSH) client1 "sudo make -C /root/samba-integration-tests tests"
```

Now that #50 is merged, and has a changed target name, rename it here:
```suggestion
	@$(VSSH) client1 "sudo make -C /root/samba-integration-tests test"
```